docs/core/index.md

```diff
@@ -1,47 +0,0 @@
----
-title: .NET Core guide
```

Comment:
Rather than just delete this article, can we incorporate its content into the "About" article?

Reply:
The About article goes more in depth, so I created a new "introduction" article with all the info from the old index.md article.